Dockerfile

```diff
@@ -73,5 +72,7 @@ RUN \
     openssl-devel mariadb-devel sendmail curl wget help2man perl-libwww-perl && \
     yum clean all && \
     rm -rf /var/cache/yum/* && \
+    sed -i '/session    required   pam_loginuid.so/c\#session    required   pam_loginuid.so' /etc/pam.d/crond && \
```

Comment:
I'm curious why you comment out this section.  Was there a specific problem you were trying to resolve?

Reply:
Before switching to root I had an issue running as apache too. https://stackoverflow.com/questions/21926465/issues-running-cron-in-docker-on-different-hosts Don't remember exact circumstances, so might not be needed anymore

Reply:
Thanks for the additional info.  I can confirm the poller ran for me after you removed this change.